index.html

```diff
@@ -15,14 +15,254 @@
   <link href="https://fonts.googleapis.com/css?family=Gloria+Hallelujah|Varela+Round" rel="stylesheet">
   <!-- main css -->
   <link rel="stylesheet" type="text/css" href="css/style.css" />
-  <!-- Font -->
-  <link href="https://fonts.googleapis.com/css?family=Muli:400,700" rel="stylesheet">
   <!-- Animate CSS -->
   <link rel="stylesheet" href="css/animate.css">
-
 </head>
   <body>
+    <!-- START NAV AND BANNER FEAT#12, BUILD BY ZUBAIR -->
+    <!-- start feat: nav -->
+    <nav class="navbar navbar-expand-lg sticky-top nav-header">
+      <div class="container">
+        <!-- strat logo -->
+        <a class="navbar-brand" href="#">
+          <img src="img/logo.png" alt="Lunatic">
+        </a>
+        <!-- end logo -->
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <!-- start nav items -->
+        <div class="collapse navbar-collapse" id="navbarText">
```

Comment:
Bug: No vertical scrolling for collapse menu. Scroll bar scrolls website contents.

Reply:
@marcin-wicha I need your help on this. can you help me to solve this bug?

Reply:
@a2-zubair I'm not sure how to fix this one. I propose to add it as a separate issue and solve in last sprint. We can work together on it then.

Reply:
@marcin-wicha ok cool.